conf/ConfigureFromEnv.php

```diff
@@ -139,7 +142,12 @@
 }
 
 if(defined('SS_ERROR_LOG')) {
-	SS_Log::add_writer(new SS_LogFileWriter(BASE_PATH . '/' . SS_ERROR_LOG), SS_Log::WARN, '<=');
+	$logger = Injector::inst()->get('Logger');
+	if($logger instanceof Logger) {
+		$logger->pushHandler(new StreamHandler(BASE_PATH . '/' . SS_ERROR_LOG, Logger::WARN));
```

Comment:
Let's pick this up as part of a more complete refactor of `ConfigureFromEnv.php` into yaml config.